README.md

```diff
@@ -113,3 +117,11 @@ yarn
 ```
 
 Then commit the changed `package.json` and `yarn.lock` files so that the official build pipeline uses the new version.
+
+### Verify symlink exists between code-dot-org repo and ML playground
+In main repo: 
+ ``` 
+ cd apps/node_modules/@code-dot-org 
+ ls -l 
+ ```
+In the output, you should see: ``` ml-playground -> ../../../../../.config/yarn/link/@code-dot-org/ml-playground ```
```

Comment:
This could vary slightly based on the user's directory structure, but "something like" this should be adequate.

Reply:
Ah, good point. Updated!